plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -422,10 +451,38 @@ class NewCreateServiceModalForm extends Component {
     ];
   }
 
+  /**
+   * This function filters the error list in order to keep only the
+   * errors that should be displayed to the UI.
+   *
+   * @returns {Array} - Returns an array of errors that passed the filter
+   */
+  getUnmutedErrors() {
+    const {errors, showAllErrors} = this.props;
+    const {editedFieldPaths, editingFieldPath} = this.state;
+
+    return errors.filter(function (error) {
+      const errorPath = error.path.join('.');
+
+      // Always mute the error on the field we are editing
+      if ((editingFieldPath != null) && (errorPath === editingFieldPath)) {
+        return false;
+      }
+
+      // Never mute fields in the CONSTANTLY_UNMUTED_ERRORS fields
+      const isUnmuted = CONSTANTLY_UNMUTED_ERRORS.some(function (rule) {
+        return rule.exec(errorPath);
```

Comment:
Please use `RegExp.prototype.test()` if you only need to know wether there was a match or not.